packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.html

```diff
@@ -31,6 +31,12 @@
             ></h2>
 
             <p data-translate="privateDatabase_configuration_alert"></p>
+            <a
+                data-translate="privateDatabase_configuration_documentation"
+                href="{{:: configurationsCtrl.configuration_guide }}"
```

Comment:
**suggestion:** We can add `target="_blank"` and `rel="noopener"`

Reply:
Applied